### PR TITLE
Added html component 

### PR DESCRIPTION
**Pull request by Alek99**

### CODE REVIEW

File: pynecone/components/layout/box.py [hunk 1]
```diff
@@ -31,3 +33,38 @@ def _render(self) -> Tag:
...
+
+    @classmethod
+    def create(cls, *children, **props):
+        """Create a html component.
+
+        Args:
+            *children: The children of the component.
+            **props: The props to pass to the component.
+
+        Returns:
```
Comment (picklelo) [inline pynecone/components/layout/box.py:56]: Update this docstring

File: pynecone/components/layout/box.py [hunk 2]
```diff
@@ -31,3 +33,38 @@ def _render(self) -> Tag:
...
+            The datable html.
+
+        Raises:
```
Comment (picklelo) [inline pynecone/components/layout/box.py:59]: And here also

File: pynecone/components/layout/box.py [hunk 3]
```diff
@@ -31,3 +33,38 @@ def _render(self) -> Tag:
                 }
             )
         )
+
+
+class Html(Box):
```
Comment (picklelo) [inline pynecone/components/layout/box.py:38]: Let's separate this out to its own file

**Review by picklelo — CHANGES_REQUESTED** [review]
Sweet! Just a couple nits

**Comment by FHU-yezi** [comment]
> Sweet! Just a couple nits

But the reward is not just a bit :sweet:

**Review by picklelo — APPROVED** [review]
Nice work

**Pull request merged**